awx/ui_next/src/components/Lookup/InventoryScriptLookup.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useCallback, useEffect } from 'react';
```

Comment:
We recently removed InventoryScripts here: https://github.com/ansible/awx/pull/7577 
This file probably snuck in during a rebase, so we should remove it.